pkg/sfu/buffer/rtpstats_receiver.go

```diff
@@ -227,7 +227,7 @@ func (r *RTPStatsReceiver) Update(
 		gapTS := int64(resTS.ExtendedVal - resTS.PreExtendedHighest)
 
 		// it is possible that sequence number has rolled over too
-		if gapSN < 0 && gapTS > 0 {
+		if gapSN < 0 && gapTS > 0 && payloadSize > 0 {
```

Comment:
This was tripping on synthesised padding packet across relay, i. e. RTP header was using time stamp of the packet with payload and that was used to synthesise a dropped packet as a padding packet which had a sequence number lesser than the highest. That resulted in `gapSN` being negative and `gapTS` being positive.